docs/content/preview/secure/enable-authentication/ysql-login-profiles.md

```diff
@@ -0,0 +1,209 @@
+
+```sh
+./bin/yugabyted start --tserver_flags="ysql_enable_profile=true"
+```
+
+### Start YB-TServer services
+
+To enable login profiles in deployable YugabyteDB clusters, you need to start your YB-TServer services using the `--ysql_enable_profile` flag. Your command should look similar to the following:
+
+```sh
+./bin/yb-tserver \
+  --tserver_master_addrs <master addresses> \
+  --fs_data_dirs <data directories> \
+  --ysql_enable_auth=true \
+  --ysql_enable_profile=true \
+  >& /home/centos/disk1/yb-tserver.out &
+```
+
+You can also enable YSQL login profiles by adding the `--ysql_enable_profile=true` to the YB-TServer configuration file (`tserver.conf`).
+
+For more information, refer to [Start YB-TServers](../../../deploy/manual-deployment/start-tservers/).
+
+### Recover from complete lockout
+
+If you lock out all roles including administrator roles, you must restart the cluster with the `--ysql_enable_profile` flag disabled.
+
+While disabling login profiles allows users back in, you won't be able to change any profile information, as profile commands can't be run when the profile flag is off.
+
+To re-enable accounts, do the following:
+
+1. Restart the cluster without profiles enabled.
+1. Create a new superuser.
+1. Restart the cluster with profiles enabled.
+1. Connect as the new superuser and issue the profile commands to unlock the accounts.
+
+<!--## Timed locked behaviour
+
+A profile can be locked indefinitely or for a specific interval. The `pg_yb_role_profile` has two states for the different LOCK behaviour:
+
+- L (LOCKED): Role is locked indefinitely.
+- T (LOCKED(TIMED)): Role is locked for a specified duration.
+
+A role is moved to the LOCKED(TIMED) state when the number of consecutive failed attempts exceeds the limit. The interval (in seconds) to lock the role is read from `pg_yb_profile.prfpasswordlocktime`. The interval is added to the current timestamp and stored in `pg_yb_role_profile.pg_yb_rolprflockeduntil`. Login attempts by the role before `pg_yb_role_profile.pg_yb_rolprflockeduntil` will fail. If the column is NULL, then it is moved to LOCKED state instead.
+
+When the role successfully logs in after `pg_yb_role_profile.pg_yb_rolprflockeduntil`, the role is moved to the OPEN state, and is allowed to log in. Failed attempts after the lock time out period don't modify `pg_yb_role_profile.pg_yb_rolprflockeduntil`. -->
+
+## Manage login profiles
+
+When profiles are enabled, you can manage login profiles using the following commands:
+
+- CREATE PROFILE
+- DROP PROFILE
+- ALTER ROLE
+
+Only superusers can create or drop profiles, and assign profiles to roles.
+
+To create a profile, do the following:
+
+```sql
+CREATE PROFILE myprofile LIMIT
+  FAILED_LOGIN_ATTEMPTS <number>;
+  [PASSWORD_LOCK_TIME <days>];
+```
+
+Note that PASSWORD_LOCK_TIME is optional, and timed locking is not currently implemented.
+
+To drop a profile, do the following:
```

Comment:
To drop a profile, do the following:  >  You can drop a profile as follows:  (suggestion)